docs/docs/CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## v0.9.6
+
+This is a bug fix release.  
+
+Issues addressed include Istio support and non-standard port handling.
```

Comment:
Might be worth linking to the issue here.

Reply:
👍